### PR TITLE
webgl-compressed-texture-s3tc-srgb.html

### PR DESCRIPTION
**Pull request by kainino0x**
This test is for the currently-draft WEBGL_compressed_texture_s3tc_srgb extension.

To review this, you might want to look at only the second commit by itself.

### CODE REVIEW

File: sdk/tests/conformance/extensions/webgl-compressed-texture-s3tc-srgb.html [hunk 1]
```diff
@@ -636,6 +648,17 @@
     }
     return wtu.makeImageFromCanvas(c);
 }
+
+function sRGBChannelToLinear(value) {
```
Comment (kenrussell) [inline sdk/tests/conformance/extensions/webgl-compressed-texture-s3tc-srgb.html]: Could you add comment about where this formula comes from?

**Comment by kenrussell** [comment]
Excellent work! LGTM overall.

Could you please add the new test to 00-test-list.txt?

One small comment. Would you mind addressing it as well?

**Comment by kainino0x** [comment]
Done.

File: sdk/tests/conformance/extensions/00_test_list.txt [hunk 2]
```diff
@@ -28,6 +28,7 @@ webgl-debug-shaders.html
 --min-version 1.0.3 webgl-compressed-texture-atc.html
 --min-version 1.0.3 webgl-compressed-texture-pvrtc.html
 --min-version 1.0.2 webgl-compressed-texture-s3tc.html
+--min-version 1.0.2 webgl-compressed-texture-s3tc-srgb.html
```
Comment (kenrussell) [inline sdk/tests/conformance/extensions/00_test_list.txt]: --min-version 1.0.4 (top of tree version in development) at this point. This is only really for bookkeeping, but let's keep things consistent.

**Comment by kenrussell** [comment]
Sorry, tiny minor nit.

**Comment by kainino0x** [comment]
Okay, got it.

**Comment by kenrussell** [comment]
Excellent. LGTM

**Pull request merged**